src/lib/components/tooltip/StackingWord.svelte

```diff
@@ -1,21 +1,27 @@
-<script>
+<script lang="ts">
   import { getContext } from "svelte";
   import transientBooleanStore from "$lib/util/transient-boolean-store";
-  const callbacks = getContext("rill:app:ui:shift-click-action-callbacks");
-  let shiftClicked = transientBooleanStore();
   export let isStacked = false;
+  export let key: "command" | "shift";
+
+  let keyCallbacks;
+  if (key === "command") {
+    keyCallbacks = getContext("rill:app:ui:command-click-action-callbacks");
+  } else if (key === "shift") {
+    keyCallbacks = getContext("rill:app:ui:shift-click-action-callbacks");
+  }
```

Comment:
I very much agree that this isn't very elegant, but refactoring these actions to use one context is a bit of a rabbit hole, and I made the conscious decision to incur some tech debt here to get this done quickly. I'd vote for doing that refactor another time.

Reply:
I don't think this is enough of a big deal to block on it, so I'm going to go ahead and mark this PR approved @ericpgreen2. But I would still recommend spending the cycles to avoid adding extra technical debt at this time. First, I don't think it will be that much of a rabbit hole; second, yes we are looking towards the July release, but I don't think we're in such dire time crunch yet that we can't spend cycles on code quality as we go; and third I'll offer my general perspective, which you can take for whatever it's worth, and even if you don't want to follow it now (which would be fine), maybe it's worth saying for later :-)

(I'm sure you've heard this perspective from other people before, because it's a boring and commonplace perspective; and I expect you will have guessed that this is my approach by now, but since we're still getting to know each other's working styles, I thought I'd go ahead an write it out :-) )

Little bits of tech debt like this begin accruing interest as soon as they are merged, and this issue you're working on and it's surrounding code will probably never be as clear to you as they are right now when you are fresh on this part of the code -- and they will certainly be less clear to someone else in 3 months when they look at this code and wonder: "oh geez, why was this nearly identical code copy/pasted? why are there two contexts? was there some subtle nuance in here that required that approach? some state that absolutely must be managed separately? some side effect that each of these units of copy/pasted has on the rest of the system that prevents them being abstracted? is this even safe to touch?" So, agreed, it may be a bit of extra work today... but it may be a 1.5x extra work for you later, or 3x work for someone else later.

Reply:
That's all fair :). I just added a comment in `StackingWord.svelte` to clarify that it's tech debt, so we don't take the decision as gospel.